pkg/manager/generator_preview.go

```diff
@@ -59,17 +59,22 @@ func (g *PreviewGenerator) Generate() error {
 
 	encoder := ffmpeg.NewEncoder(instance.FFMPEGPath)
 
-	if err := g.generateConcatFile(); err != nil {
+	tmpFiles, err := g.generateConcatFile()
```

Comment:
The concat file is only used in `generateVideo` and `generateConcatFile` is not expensive.

I think the `generateConcatFile` should be moved into `generateVideo`, just before `ScenePreviewVideoChunkCombine`. 

`generateVideo` should have a local `generatedFiles` string slice, and is appended to as it generates each file. Use a deferred function to remove the files. This way, only actually created files will be deleted (you won't end up with delete errors for ungenerated files).